vavr/src/main/java/io/vavr/collection/Array.java

```diff
@@ -1320,21 +1331,14 @@ public int length() {
     }
 
     @Override
-    public Array<T> takeUntil(Predicate<? super T> predicate) {
-        Objects.requireNonNull(predicate, "predicate is null");
-        return takeWhile(predicate.negate());
+    public Array<T> takeRightUntil(Predicate<? super T> predicate) {
+        return io.vavr.collection.Collections.takeRightUntil(this, predicate);
```

Comment:
Okay. I was wondering because in the existing code I usually saw `io.vavr.collection.Collections` fully qualified.